WordPressKit/ReaderPostsEnvelope.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
```

Comment:
This import might be unnecessary.

Reply:
Yes, that's true for the current project setup.  But I think it's needed for SPM support.